### PR TITLE
Issue/babel fix and socket connections

### PR DESCRIPTION
**Pull request by doncesarts**
fixes socket connections and babel config

### CODE REVIEW

File: packages/scrum-client/src/hooks/ClientSocketEvents.js [hunk 1]
```diff
@@ -41,6 +41,7 @@ const ClientSocketEvents = (ioUri = API_CONSTANTS.API_BASE_URL) => {
   const onStoryVotesUpdate = cb => {
     console.log('onStoryVotesUpdate');
     socket.on(EVENT.STORY_VOTES_UPDATE, data => {
+      console.log('onStoryVotesUpdate-data', data);
```
Comment (carlospatinos) [inline packages/scrum-client/src/hooks/ClientSocketEvents.js:44]: Is it intended to keep this console? Or was it just for testing? I am fine with it just double checking

File: packages/scrum-server/src/services/socket/ServerSocketState.js [hunk 2]
```diff
@@ -46,7 +46,7 @@ const ServerSocketState = (initialState = []) => {
     logger.debug(`setRoomStoryVote on room {${room.id}} for user {${userId}} with value {${vote}}`);
     const _room = rooms.get(room.id);
     _room.storyVotes.set(userId, vote);
-    logger.silly(`setRoomStoryVote on room {${room.id}} for user {${userId}} has {${JSON.stringify(_room.storyVotes)}}`);
+    logger.silly(`setRoomStoryVote on room {${room.id}} for user {${userId}} has {${JSON.stringify(Array.from(_room.storyVotes.entries()))}}`);
```
Comment (carlospatinos) [inline packages/scrum-server/src/services/socket/ServerSocketState.js:49]: I think at some point we may need to improve this. I think besides this is silly level cpu is still consumed due to the transformation of the data before the log is even called. We will need a strategy anyway but all good here

**Review by carlospatinos — APPROVED** [review]

File: packages/scrum-server/src/services/socket/ServerSocketState.js (hunk 2 above)
Comment (doncesarts) [reply packages/scrum-server/src/services/socket/ServerSocketState.js:49]: socket state  is not meant to be in memory ....  i would imagine this needs to be ported to mongodb, right ?

File: packages/scrum-client/src/hooks/ClientSocketEvents.js (hunk 1 above)
Comment (doncesarts) [reply packages/scrum-client/src/hooks/ClientSocketEvents.js:44]: to leave it until we don have more issues, then we can remove , but for now it is intended

File: packages/scrum-server/src/services/socket/ServerSocketState.js (hunk 2 above)
Comment (carlospatinos) [reply packages/scrum-server/src/services/socket/ServerSocketState.js:49]: right

**Review by carlospatinos — APPROVED** [review]

**Pull request merged**